db/db-test/db.test.js

```diff
@@ -53,8 +36,6 @@ describe('database helpers - save and find', () => {
   });
 
   afterAll((done) => {
```

Comment:
nice, did this actually end the connection when you run the tests?

Reply:
No, I thought it did but turns out it doesn't. will refactor with next push